recipe/meta.yaml

```diff
@@ -19,45 +19,54 @@ outputs:
     script: build_base.bat  # [win]
     script: build_base.sh  # [not win]
     build:
-      skip: true  # [py<37]
+      skip: true  # [py<38]
       ignore_run_exports:
         - tk  # [linux]
+        - zlib
     requirements:
       build:
+        - python
+        - cross-python_{{ target_platform }}       # [build_platform != target_platform]
+        - numpy
```

Comment:
Is numpy needed here when we have it in `host`? If so, why do we not have the same pinnings like we do in host?

Reply:
Hey Marco! So the rationale for the pinnings is our artifacts for our different platforms run on different versions of numpy so it's best to use two different versions in the host section. [See here ](https://github.com/AnacondaRecipes/matplotlib-feedstock/pull/13#issuecomment-1296704563) for further explanation.

Reply:
This doesn't answer my questions. Let me rephrase:

* Why do we need numpy in build and in host at the same time?
* Why are the build pinnings different than in host?

Reply:
> python and numpy should be removed from host. 

From `build` **not** from `host`

Reply:
@skupr-anaconda yes, thank you!

Reply:
Went ahead and removed `# [build_platform != target_platform]` , `python`, and `numpy` from the build section.